src/support/components/FeedbackQuestionsOverlay.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {FC, ChangeEvent, useContext, useState} from 'react'
+
+// Components
+import {
+  Button,
+  ButtonType,
+  ComponentColor,
+  ComponentStatus,
+  Form,
+  Overlay,
+  TextArea,
+} from '@influxdata/clockface'
+
+// Contexts
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+
+// Types
+import ErrorBoundary from 'src/shared/components/ErrorBoundary'
+
+interface OwnProps {
+  onClose: () => void
+}
+
+const FeedbackQuestionsOverlay: FC<OwnProps> = () => {
+  const {onClose} = useContext(OverlayContext)
+  const [input, setInput] = useState('')
```

Comment:
`input` is a bit of an overloaded term. it could refer to an html text input in a form, or a concept of input. I think it's generally better to name variables after the concept it represents in the UI. `description` is what the label is for the field, so that's a start. it might aslo be called something like `feedbackText` for example - both of these things are what the text input represents; `description` is a little more vague than `feedbackText`, and in general, I find clarity in variable names to be easier to follow and reason about.